src/components/pages/Author.vue

```diff
@@ -43,16 +44,37 @@ export default {
     CardCollection
   },
   props: {
+    backendUrl: '',
     platform: ''
   },
   data: function () {
     return {
-      'urlArguments': ''
+      'urlArguments': '',
+      author: {
+        'id': 0,
+        'name': '',
+        'cards': []
+      }
+    }
+  },
+  methods: {
+    get_author: function (id) {
+      var that = this
+      window.$.getJSON(this.backendUrl + '/dev/author/id/' + id, function (j, s) {
```

Comment:
Just change this to the new format with vue-resources , you can see it in any other component that fetches from the backend

Reply:
Good catch, this is fixed now.